legistar/bills.py

```diff
@@ -232,7 +232,31 @@ def matters(self, since_datetime=None):
         params = {'$orderby': 'MatterLastModifiedUtc'}
 
         if since_datetime:
-            params['$filter'] = "MatterLastModifiedUtc gt datetime'{since_datetime}'".format(since_datetime=since_datetime.isoformat())
+            update_fields = ('MatterLastModifiedUtc',
+                             'MatterIntroDate',
+                             'MatterPassedDate',
+                             'MatterDate1',
+                             'MatterDate2',
+                             # 'MatterEXDate1', # can't use all 17 search
+                                                # terms, this one always
+                                                # seems to be not set
+                             'MatterEXDate2',
+                             'MatterEXDate3',
+                             'MatterEXDate4',
+                             'MatterEXDate5',
+                             'MatterEXDate6',
+                             'MatterEXDate7',
+                             'MatterEXDate8',
+                             'MatterEXDate9',
+                             'MatterEXDate10',
```

Comment:
Do we have any idea what these are? I think `MatterEXDate10` is the last time the matter was synced with SmartLogic, for instance. I suppose the idea is that we're ok with a few false positives, so long as we are reducing false negatives...

Reply:
i have no idea what these are. They are also set in Chicago, so it's probably not just SmartLogic.

I just added everything that looked like a Date with the idea that false positives are okay.